lib/Stampie/Mailer.php

```diff
@@ -173,4 +200,54 @@ protected function buildIdentityString($identities)
     {
         return IdentityUtils::buildIdentityString($identities);
     }
+
+    /**
+     * Take a Message and return a Stampie Response
+     *
+     * @param MessageInterface $message
+     *
+     * @return Response
+     */
+    private function doSend(MessageInterface $message)
+    {
+        $content = $this->format($message);
+        $headers = $this->getHeaders();
+        $files = $this->getFiles($message);
+
+        if (!empty($files)) {
+            // HTTP query content
+            parse_str($content, $fields);
+            $builder = new MultipartStreamBuilder();
+
+            foreach ($fields as $name => $contents) {
+                if (!is_array($contents)) {
```

Comment:
Removing the `!` and flipping the conditions would make it more read able, also you can remove the `else` part of the conditional by using `continue`.